lib/facter/facts/freebsd/is_virtual.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Facts
+  module Freebsd
+    class IsVirtual
+      FACT_NAME = 'is_virtual'
+
+      def initialize
+        @virtual = Facter::VirtualDetector.new
+      end
+
+      def call_the_resolver
+        fact_value = @virtual.platform
+
+        Facter::ResolvedFact.new(FACT_NAME, check_if_virtual(fact_value))
+      end
+
+      private
+
+      def check_if_virtual(found_vm)
+        Facter::FactsUtils::PHYSICAL_HYPERVISORS.count(found_vm).zero?.to_s
```

Comment:
is_virtual should be boolean, please drop the `to_s` method. (there is a bug regarding this inconsistency in fact types in the Linux `is_virtual` fact as well)

Reply:
I added a commit to fix this.  While here I rebased these changes on top of master.